assets/sass/modules/_enroll.scss

```diff
@@ -416,7 +413,35 @@ $enroll-content-spacing: 15px;
   }
 }
 
-.enroll-manual-push {
+.enroll-manual-push ,
+.enroll-manual-totp {
+  .secret-instructions {
```

Comment:
nit: rename to `secret-key-instructions`, as in, this section is about a secret key, not about some secret instructions that nobody should know ;)